emailserver.js

```diff
@@ -45,21 +45,23 @@ var server = smtp.createServer(HOSTNAME, function (req) {
 
     mailparser.on('end', function(mail) {
       mail.receivedAt = new Date().toISOString();
-      var user = req.to;
-      log('Received message for', user);
+      var users = req.to;
+      log('Received message for', users);
```

Comment:
This rings a bell. I have a refresh branch to move this to node4 (from about a year ago) and I had to fix this problem. Must follow through with that branch (which includes a load test).